pkg/operator/defragcontroller/defragcontroller.go

```diff
@@ -150,7 +150,9 @@ func (c *DefragController) checkDefrag(ctx context.Context, recorder events.Reco
 			recorder.Eventf("DefragControllerDefragmentAttempt", "Attempting defrag on member: %s, memberID: %d, dbSize: %d, dbInUse: %d, leader ID: %d", member.Name, member.ID, status.DbSize, status.DbSizeInUse, status.Leader)
 			if _, err := c.etcdClient.Defragment(ctx, member); err != nil {
 				// Defrag can timeout if defragmentation takes longer than etcdcli.DefragDialTimeout.
-				errors = append(errors, fmt.Errorf("failed to defragment etcd member: %q :%v", member.Name, err))
+				errMsg := fmt.Sprintf("failed defrag on member: %s, memberID: %d: %v", member.Name, member.ID, err)
```

Comment:
Nit: member IDs are hex right? Or if all our logs/events are using base-10 then we can probably keep it the same.
```suggestion
				errMsg := fmt.Sprintf("failed defrag on member: %s, memberID: %x: %v", member.Name, member.ID, err)
```

Reply:
good call

Reply:
etcd for example

```
2021-11-16T15:31:32.863903696Z {"level":"info","ts":"2021-11-16T15:31:32.863Z","caller":"rafthttp/stream.go:249","msg":"set message encoder","from":"c15516da369193df","to":"1a1a2c3fa1c72d72","stream-type":"stream Message"}
2021-11-16T15:31:32.863903696Z {"level":"info","ts":"2021-11-16T15:31:32.863Z","caller":"rafthttp/stream.go:249","msg":"set message encoder","from":"c15516da369193df","to":"cf5e639dad63a635","stream-type":"stream MsgApp v2"}
2021-11-16T15:31:32.863903696Z {"level":"info","ts":"2021-11-16T15:31:32.863Z","caller":"rafthttp/peer_status.go:53","msg":"peer became active","peer-id":"1a1a2c3fa1c72d72"}
2021-11-16T15:31:32.863903696Z {"level":"info","ts":"2021-11-16T15:31:32.863Z","caller":"rafthttp/peer_status.go:53","msg":"peer became active","peer-id":"cf5e639dad63a635"}
```